includes/wccom-site/rest-api/endpoints/class-wc-rest-wccom-site-installer-controller.php

```diff
@@ -95,6 +95,11 @@ public function get_install_state( $request ) {
 	 * @return bool|WP_Error
 	 */
 	public function install( $request ) {
+		$requirements_met = WC_WCCOM_Site_Installer_Requirements_Check::met_requirements();
```

Comment:
Is this only on the `POST` request right now?

Can we add it to the `GET` request also, so there's a lightweight way to check? This allows us to use the `GET` api as a preflight check before allowing the flow (as we originally hoped). I'd suggest we just add this check to all API methods, unless there's a major issue with that (e.g. performance).

Reply:
@haszari  - I have added the validation to `GETm /installer` endpoint in 4fabaed

Reply:
Tested this today, and things are working well :)

- With cron enabled, I was able to auto-install.
- With cron disabled (added `define( 'DISABLE_WP_CRON', true );` to `wp-config.php`), auto install failed after the first site-api call.